### ISSUE TITLE
scrollbar jumping

### ISSUE THREAD

**Issue by BetterCallSky** [henrybuilt/react-sticky-table#42]
See video 
https://penegue-gmail.tinytake.com/sf/MjY2NTU0MF83OTk5OTAz
The scrollbar is jumping, when there are no interactions. Testing the demo here https://henrybuilt.github.io/react-sticky-table/ .

```
Google Chrome    66.0.3359.181 (Official Build) (64-bit) (cohort: Stable)
Revision    a10b9cedb40738cb152f8148ddab4891df876959-refs/branch-heads/3359@{#828}
OS    Windows
```

**Comment by maxhudson** [comment]
@BetterCallSky yikes. first time I've heard of this. I'll see if I can figure it out - I have a hunch about the source. thanks for bringing it up.

**Comment by emirdeliz** [comment]
I have the same problem after change zoom of the browser. I'm using MacOs

**Comment by maxhudson** [comment]
resolved in v3.0.1
